src/constants/route.ts

```diff
@@ -1,81 +1,42 @@
-export interface PageMetadata {
-  name: string;
-  path: string;
-  description?: string;
-  ogImg?: string;
-  twitterImg?: string;
-}
-
-export const DEFAULT_METADATA = {
+const DEFAULT_METADATA = {
   name: "",
+  title: "Level Up",
   description:
     "The best platform for Web3 builders to skill up, find jobs and launch ideas",
   ogImg: "/images/level-up-preview.png",
```

Comment:
remove `ogImg` and `twitterImg` because `opengraph-image.ong` and `twitter-image.png`  are working now